karma.conf.js

```diff
@@ -16,6 +16,13 @@ module.exports = function(config) {
   config.set({
     // base path, that will be used to resolve files and exclude
     basePath: '',
+    plugins: [
+      'karma-coverage',
+      'karma-mocha',
+      'karma-ng-html2js-preprocessor',
+      'karma-sinon-chai',
+      'karma-chrome-launcher',
```

Comment:
Worth noting is that using Grunt to run karma for some reason removes the ability to run the tests in the browser, which is a deal-breaker IMO.